learningequality/studio
# Fixes for syncing invitation updates between sender and invited user

## Description
_This is work in progress on the following issues:_
- [x] Ensure invitation updates are loaded whenever navigating to the editable channels list
- [x] Make sure deleted invitations are deleted for both the sender and the invited user

#### Issue Addressed (if applicable)

Addresses #1874

## Steps to Test
Using two separate browser instances (e.g. one of them an incognito window), login as two users.
- [x] As user A, invite user B to edit a channel
- [x] As user B, check to see that this invitation showed up in your channel list page (may need to wait ~5 s)
- [x] As user B, try accepting (declining) the invitation, then check as user A that the invitation disappears
- [x] Alternatively, as user A, delete the invitation, then check to make sure that as user B:
  1, [ ] you cannot accept it
  2. [ ] repeat this case, but instead of trying to accept the invitation, wait a little while and check to see that the revoked invitation has disappeared 

## Implementation Notes (optional)
### COMING SOON

#### At a high level, how did you implement this?

*Briefly describe how this works*

#### Does this introduce any tech-debt items?

*List anything that will need to be addressed later*


## Checklist

*Delete any items that don't apply*


## Comments

*Any additional notes you'd like to add*

## Reviewers

If you are looking to assign a reviewer, here are some options:
- Jordan jayoshih (full stack)
- Aron aronasorman (back end, devops)
- Micah micahscopes (full stack)
- Kevin kollivier (back end)
- Ivan ivanistheone ([Ricecooker](https://github.com/learningequality/ricecooker))
- Richard rtibbles (full stack, [Kolibri](https://github.com/learningequality/kolibri))
- Radina @radinamatic (documentation)
